lib/person.rb

```diff
@@ -62,4 +62,21 @@ def unassign(role:, date:)
 
     @assigned_shifts.delete(shift_to_unassign)
   end
+
+  def to_h
+    excluded_ivars = ["@roles_config"]
+
+    hash = {}
+    instance_variables.each do |variable|
```

Comment:
You could make this `.each_with_object({}) do |hash, variable|` to remove the need for line 69?

Reply:
Ah, fair enough! Weird ruby internals.